Sources/Networking/Utils/FileDataWriter.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  FileWriter.swift
+//
+//
+//  Created by Jan Kodeš on 24.01.2024.
+//
+
+import Foundation
+
+public protocol FileDataWriterProtocol {
```

Comment:
Wouldn't a `FileDataWriting` be a better name since we are using the `ing` convention throughout the project? Also could you please include at least one sentence for both the protocol and the `FileDataWriter` for documentation? 🙏

Reply:
Definitely!